spotguide/spotguide.go

```diff
@@ -198,16 +202,31 @@ func (s *SpotguideManager) isSpotguideReleaseAllowed(release scm.RepositoryRelea
 	return supported && (!prerelease || viper.GetBool(config.SpotguideAllowPrereleases))
 }
 
-func (s *SpotguideManager) ScrapeSharedSpotguides() error {
-	if s.sharedLibraryOrganization == nil {
-		return fmt.Errorf("failed to scrape shared spotguides")
+func (s *SpotguideManager) ScheduleScrapingSharedSpotguides() error {
```

Comment:
Given this is called once from main and doesn't use the manager itself, I don't think it has to be a member of the manager. This could be a plain function, accepting a workflow client. Then the manager also does not have to receive a workflowClient instance in it's constructor.